packages/fether-electron/src/main/app/options/config/index.js

```diff
@@ -6,10 +6,61 @@
 import path from 'path';
 import url from 'url';
 
+import Pino from '../../utils/pino';
 import { staticPath } from '../../utils/paths';
+import cli from '../../cli';
+import {
+  DEFAULT_CHAIN,
+  DEFAULT_WS_PORT,
+  IS_PROD,
+  TRUSTED_LOOPBACK,
+  TRUSTED_WS_ORIGINS
+} from '../../constants';
+
+const pino = Pino();
+
+pino.info(
+  `Running Fether in ${IS_PROD ? 'production' : 'development'} environment`
+);
+
+/**
+ * Note: If the user provides a custom CLI port to `cli.wsPort` then
+ * we 'dynamically' trust it in addition to the `DEFAULT_WS_PORT` in
+ * fether-electron/src/main/index.js, which is where we only
+ * permit requests from trusted paths.
+ *
+ * WARNING: DO NOT add the custom CLI interface `cli.wsInterface` as a
+ * trusted host. This may avoid Fether being launched with a
+ * malicious remote `cli.wsInterface` and sending sensitive user information
+ * (i.e. account password) over RPC.
+ * See https://github.com/paritytech/fether/pull/451#discussion_r268732256
+ *
+ * Note: We also disallows users from using Fether
+ * with a remote node.
+ * WARNING: SSH tunnels from an attacker are still possible.
+ */
+const DEFAULT_HTTP_PORT = '3000';
+const CUSTOM_WS_PORT = cli.wsPort;
+const TRUSTED_HOSTS = ['api.github.com', 'github.com'];
+const TRUSTED_WS_PORTS = [DEFAULT_WS_PORT, CUSTOM_WS_PORT];
+const DEFAULT_HTTP = `http://localhost:${DEFAULT_HTTP_PORT}`;
+const DEFAULT_HTTP_TRUSTED_LOOPBACK = `http://${TRUSTED_LOOPBACK}:${DEFAULT_HTTP_PORT}`;
+const TRUSTED_URLS = [
+  DEFAULT_HTTP_TRUSTED_LOOPBACK,
+  `ws://${TRUSTED_LOOPBACK}:${DEFAULT_WS_PORT}`,
```

Comment:
we haven't setup `wss` and associated certificates yet,  as that would be a future PR, so i'll switch it back to just `Ws` instead of `WsSecure` to remove the confusion in the interim

Reply:
Makes sense